_posts/2025-09-08-data-and-florence-trends.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Data Gathering and Florence Trends"
+date: 2025-09-08
+author: "Bernardo&Martina"
+tags: [data-analysis, airbnb, florence, tourism]
+categories: [analysis]
+---
+
+# La Raccolta Dati
+
+Ogni analisi che si rispetti parte sempre da una raccolta dati o da una base dati già esistente. Dato che siamo interessati ad osservare il mercato degli affitti brevi in Italia sulla piattaforma AirBnB, la miglior risorsa attualmente disponibile è il sito [Inside Airbnb](https://insideAirbnb.com/get-the-data/).
+
+Inside AirBnb permette di scaricare un anno di dati per varie zone di interesse, con snapshot della piattaforma Airbnb ogni circa 3 mesi. I dati che è possibile scaricare comprendono i seguenti file:
+
+- **listings.csv** La lista delle strutture ricettive attive al momento dello snapshot.  
+- **calendar.csv** Le date disponibili per ciascuna struttura **TODO: Guardare meglio questi dati e capire cosa sono**  
+- **reviews.csv** Lo storico di tutte le recensioni di ciascuna struttura.  
+- **neighbourhoods.geojson** Una divisione in quartieri o comuni dell'area di interesse.  
+
+Considerando i dati italiani, non sembra esserci una ratio precisa dietro la raccolta presente in Inside Airbnb. Sono presenti varie città importanti dal punto di vista turistico e non (e.g. Roma, Milano, Venezia, Firenze, Napoli), ma anche intere regioni non suddivise in città o aree metropolitane (e.g. Sicilia). 
+
+Abbiamo scaricato tutti i dati riguardanti l'Italia al momento e iniziato l'esplorazione dei dati per capirne meglio la struttura. L'idea è quella di avere uno storico sufficientemente lungo da poter osservare cambiamenti su periodi di vari anni. Per la policy stessa di Inside Airbnb, non ci è possibile condividere i dati che collezioniamo, ma potremo diffondere le analisi ed eventualmente gli script senza problemi.
+
+Inside Airbnb non fornisce una documentazione approfondita dei vari campi presenti in ogni file e nella suddivisione spaziale di ogni area. In un prossimo post andremo a descrivere un po’ meglio quello che si trova nei vari file e come estrarre qualche informazione interessante. **NOTA: che ne pensi marti?**
```

Comment:
@bernomone allora Berna io tutta questa parte la toglierei perché secondo me facciamo meglio a fare un post a parte su come sono strutturati i dati e come li estraiamo?

Reply:
se sei d'accordo su questo tiro fuori questa parte dal testo e la metto in un issue per un nuovo post